### PR TITLE
changes

### PR DESCRIPTION
**Pull request by Mateusz3124**

### CODE REVIEW

**Comment by Mateusz3124** [comment]
nice

**Review by Mateusz3124 — COMMENTED** [review]
nice changes

**Review by Mateusz3124 — COMMENTED** [review]
nice changes

**Review by MatSobol — APPROVED** [review]
nice

**Review by MatSobol — APPROVED** [review]
this is review

File: WebApi/Program.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@
     using (StreamReader reader = new StreamReader("version"))
     {
         string line = reader.ReadLine();
-        Console.WriteLine(line);
+        Console.WriteLine("The current version of app is:" + line);
```
Comment (MatSobol) [inline WebApi/Program.cs:20]: fsfsdfsd

Comment (MatSobol) [reply WebApi/Program.cs:20]: nice

**Review by MatSobol — APPROVED** [review]
why not working xd

**Review by MatSobol — APPROVED** [review]

**Pull request merged**